### ISSUE TITLE
Vent light question

### ISSUE THREAD

**Issue by ryandcole** [CountDeMonet/ArduinoProtonPack#3]
This looks awesome.  What neopixels did you use for the venting lights on the pack and wand?

**Comment by CountDeMonet** [comment]
Thanks. This is the set I bought. Have a few left over still :)

https://www.amazon.com/gp/product/B01AG8X1X8

**Comment by ryandcole** [comment]
Awesome.  Thank you!  Starting to plan the build now with the Spirit pack and updated 3D printed wand.  Hope to have it ready for Halloween!  Thanks!

**Comment by CountDeMonet** [comment]
Good luck. I really enjoyed this build. Happy to answer any questions that come up

**Comment by ryandcole** [comment]
Awesome.  I'm sure that I'll have them!  Below are the goggles I made for last Halloween.
<img width="592" alt="screen shot 2018-06-13 at 4 27 03 pm" src="https://user-images.githubusercontent.com/15256658/41383655-a11cf20c-6f26-11e8-98d4-acb9ee7438fb.png">

**Comment by acfq48** [comment]
What wire did you use count?  It's triple stranded and nicely kept
together.  Mine kinda looks like a hodgepodge of 24 gauge garbage...

**Comment by CountDeMonet** [comment]
Those goggles look great. Nice work. 

I used a few different kinds of wire. For the wires that are connected I used this ribbon wire

Uxcell a16030200ux0389 Ribbon IDC Cable Wire Rainbow Cable Flat Color Rainbow, 2 m, 6.6', 40 Way, 40 Pin https://www.amazon.com/dp/B01JRA853W/ref=cm_sw_r_cp_taa_i_v3AiBbWW0KRZ4

I also used it for the pack ribbon wire that is displayed on the outside.

I like this stuff for pretty much everything else. Very flexible and strong 

Striveday&trade;Flexible Silicone Wire 24awg Electric wire 24 gauge Coper Hook Up Wire 300V Cables electronic stranded wire cable electrics DIY BOX-1 https://www.amazon.com/dp/B01LH1G2IE/ref=cm_sw_r_cp_taa_i_t6AiBbMJKFZTC
